string name
string type
---
float64[] pos
float64[] rot
